File: 241122/특정 숫자 맞추기/catching-specific-number.c
#include <stdio.h>

int main() {
    // 여기에 코드를 작성해주세요.

    while(1) {
        int n;

        scanf("%d", &n);

        if (n<25) {
            printf("Higher\n");
        }
        else if(n>25){
            printf("Lower\n");
        }
        else if(n==25){
            printf("Good\n");
            break;
        }
    }
    return 0;
}



// n<25 h
// n>25 L
//n==25 G

File: 240703/특정 숫자 맞추기/catching-specific-number.py
#무한 loop --> whle True:
while True:
    n = int(input())
    if(n < 25):
        print("Higher")
    elif(n>25):
        print("Lower")
    else:
        print("Good")
        # 무한 루프 탈출 시 --> break
        break